browser/extensions/api/brave_shields_api.cc

```diff
@@ -23,8 +23,12 @@
 #include "chrome/browser/extensions/chrome_extension_function_details.h"
 #include "chrome/browser/extensions/extension_tab_util.h"
 #include "chrome/browser/profiles/profile.h"
+#include "chrome/common/chrome_features.h"
```

Comment:
you need a dep for `//chrome/common` in brave/browser/extensions/BUILD.gn

Reply:
I verified it's there now. Resolving this.